atomic_physics/tests/test_mg25_consts.py

```diff
@@ -0,0 +1,60 @@
+"""
+References:
+[1] - R. Srinivas, Laser-free trapped-ion quantum logic with a radiofrequency
+        magnetic field gradient, PhD Thesis, University of Colorado (2016)
+"""
+
+import unittest
+from atomic_physics.ions import mg25
+from atomic_physics.utils import field_insensitive_point
+import numpy as np
+
+S12_41_31_clock_field = 212.8e-4  # [1]
```

Comment:
This variable name seems misleading if it's actually the |3, 1> <-> |2, 1> transition you are considering.